common/element_handle.go

```diff
@@ -270,7 +270,10 @@ func (h *ElementHandle) fill(_ context.Context, value string) error {
 	if !ok {
 		return fmt.Errorf("unexpected type %T", result)
 	}
-	if ok && s != resultDone {
+
+	if s == "needsinput" {
```

Comment:
This is great. Can you put this in a constant (like `resultDone`)?

Reply:
done